requirements.txt

```diff
@@ -22,5 +22,9 @@ trio
 trustme
 uvicorn
 seed-isort-config
+charset_normalizer>=1.3,<2
+rfc3986
+certifi
+h2
```

Comment:
Were these dependencies added because we miss them for our base development install?

Reply:
Hi, When cloning the base project I was unable to use it in my py3.8 venv. theses packages were missing. running `python setup.py install` was not enough. Maybe I missed something. `model.py` require `rfc3986`. The base package fail and also the test suite.

Reply:
Did you run `scripts/install` first?

Reply:
@Ourset Yup, we have an install script that should take care of installing all package and development dependencies.

Reply:
My bad. I will revert this change.

Reply:
Done